src/univariate/continuous/maxwellboltzmann.jl

```diff
@@ -0,0 +1,84 @@
+"""
+    MaxwellBoltzmann(a)
+
+The *Maxwell-Boltzmann distribution* has probability density function
+```math
+f(x; a) = \\sqrt{\\frac{2}{\\pi}}\\frac{x^2}{a^3}\\exp\\left(-\\frac{x^2}{2a^2}\\right)
+```
+where
+```math
+a = \\sqrt{\\frac{kT}{m}}
+```
+`k` is the Boltzmann constant and `m` is the particle mass.
+
+```julia
+MaxwellBoltzmann()        # Maxwell-Boltzmann distribution with a = 1
+MaxwellBoltzmann(a)       # Maxwell-Boltzmann distribution with parameter a
+MaxwellBoltzmann(T, m)    # Maxwell-Boltzmann distribution with temperature T and mass m
```

Comment:
I suggest fixing this to some explicit unit (kcal/(mol/K), preferably):

```
MaxwellBoltzmann(T, m)    # Maxwell-Boltzmann distribution with temperature T and mass m with k in kcal mol⁻¹ K⁻¹
```

and adding a fourth method:

```
MaxwellBoltzmann(T, m, k) # Maxwell-Boltzmann distribution with temperature T and mass m given k 
```